### PR TITLE
feat(crons): Backlink to monitors from crons issue

### LINKED ISSUE #57445: Link to CRON job definition in Monitor failure issue
### Problem Statement

Currently when a Monitor failure issue is created there is no easy way to get to the CRON job defintion. This is annoying as that is the only place where the actual issue causing the monitor failure is linked (see also https://github.com/getsentry/sentry/issues/43647 I think).

We can create a webhook based alert in our internal messaging platform for the Monitor failure issue but then there is no clear next step for the developer looking at that issue. We want to define 50+ crons so it would be quite hard for them to find the particular definition of the alert they were looking for.
Can we add a link to the definition in the issue? 

### Solution Brainstorm

It should be as simple as `$PROJECT_NAME/$MONITOR_SLUG`, although I'm probably missing edge cases :sweat_smile: 

### Product Area

Crons

### PR DESCRIPTION
**Pull request by davidenwang**
Adds a `View in monitor details` button. Placement of this button is subject to a bit of discussion, but will leave this as is for now, with opportunity to relocate the button into the issue details header in the future (perhaps next to `Open in Discover`)

image

Fixes: https://github.com/getsentry/sentry/issues/57445

### CODE REVIEW

File: static/app/components/events/interfaces/crons/cronTimelineSection.tsx [hunk 1]
```diff
@@ -120,6 +136,11 @@ export function CronTimelineSection({event, organization}: Props) {
   );
 }
 
+const MonitorLinkButton = styled(LinkButton)`
+  height: 26px;
```
Comment (evanpurkhiser) [inline static/app/components/events/interfaces/crons/cronTimelineSection.tsx:140]: Damn, cc @vuluongj20

**Review by evanpurkhiser — APPROVED** [review]

File: static/app/components/events/interfaces/crons/cronTimelineSection.tsx (hunk 1 above)
Comment (davidenwang) [reply static/app/components/events/interfaces/crons/cronTimelineSection.tsx:140]: Ye not feeling super confident about stuff like this, just matched it up with the resolution selector height for now

Comment (vuluongj20) [reply static/app/components/events/interfaces/crons/cronTimelineSection.tsx:140]: We can use this — it's called `form` but it applies to both form inputs & buttons:
https://github.com/getsentry/sentry/blob/67bab6bd67f2aa3454b9f274ac1783494950b31e/static/app/utils/theme.tsx#L807-L830

Comment (vuluongj20) [reply static/app/components/events/interfaces/crons/cronTimelineSection.tsx:140]: also I thought `LinkButton` had a size prop?

Comment (davidenwang) [reply static/app/components/events/interfaces/crons/cronTimelineSection.tsx:140]: oh lol my bad for some reason thought the next smallest size from "sm" was "zero" so I styled separately, will just change to a "xs" sized button which should fix the text + height

**Review by evanpurkhiser — APPROVED** [review]
Wayyy better

**Pull request merged**